docs/Using-Virtual-Environment.md

```diff
@@ -45,9 +47,16 @@ the latest pip version using `pip3 install --upgrade pip`
 ## Windows Setup
 
 1. Create a folder where the virtual environments will reside `$ md python-envs`
-1. To create a new environment named `sample-env` execute `$ python3 -m venv python-envs\sample-env`
+1. To create a new environment named `sample-env` execute `$ python -m venv python-envs\sample-env`
 1. To activate the environment execute `$ python-envs\sample-env\Scripts\activate`
 1. Verify pip version is the same as in the __Installing Pip__ section. In case it is not the latest, upgrade to
-the latest pip version using `pip3 install --upgrade pip`
-1. Install ML-Agents package using `$ pip3 install mlagents`
+the latest pip version using `pip install --upgrade pip`
```

Comment:
The commands are inconsistent - some start with `$` and others don't. I'd lean towards not using them, especially on the windows instructions.

Reply:
Ok, I've removed `$` from the windows instructions.